samples/tab-deeplink-all-features/nodejs/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-tab-deeplink-all-features-nodejs",
+    "source": "officeDev",
+    "title": "DeepLink All Features",
+    "shortDescription": "Microsoft Teams sample app for demonstrating deeplink all features",
```

Comment:
"shortDescription": "Microsoft Teams sample app for demonstrating deep links features",

Reply:
fixed